docsearch/plugins/my-loaders/index.js

```diff
@@ -0,0 +1,19 @@
+module.exports = function(context, options) {
+  return {
+    name: 'loaders',
+    configureWebpack(config, isServer) {
+      return {
+        mode:"development",
```

Comment:
this will enforce webpack to be in dev mode even in production

Reply:
Thanks @endiliey 
Resolved by removing it.